deploy/kubernetes/rbac.yaml

```diff
@@ -58,20 +58,6 @@ roleRef:
   name: external-resizer-runner
   apiGroup: rbac.authorization.k8s.io
 
----
-# Resizer must be able to work with end point in current namespace
-# if (and only if) leadership election is enabled
```

Comment:
This comment seems wrong. It should say "leases" instead of "end point".

Reply:
https://github.com/kubernetes-csi/external-resizer/commit/15a912cfe617c0d38a84cd2b8639c2013b912358 removed the "endpoints" part without updating the comment.